apps/ai-content-generator/src/configs/features/featureConfig.tsx

```diff
@@ -12,31 +13,41 @@ enum AIFeature {
   SEO_KEYWORDS = 'seoKeywords',
 }
 
+export enum AIFeatureType {
```

Comment:
I dont understand why we need this. All you need is the key such as AIFeature.TITLE/AIFeature.CONTENT that is consumed by your Dialog component. Your base Dialog component should contain the switch-case logic and return the component to render based on that.

That base Dialog component should contain the mapping, you don't need to define any types here.